config/environments/development.rb

```diff
@@ -81,6 +81,10 @@
   config.action_mailer.delivery_method = :letter_opener
 
   config.hosts << 'bs-local.com' if ENV['OPENPROJECT_DISABLE_DEV_ASSET_PROXY'].present?
+
+  if ENV['OPENPROJECT_DEV__EXTRA_HOSTS'].present?
```

Comment:
Also this should be documented somewhere. Perhaps best in `docs/development/localhost-ssl/README.md`.

Reply:
Will do, but hopefully I can amend this soon, as I'm currently working on a rework for our docker dev env, so that we can easily launch it with TLS enabled.